### PR TITLE
added dotnet-samples

### PR DESCRIPTION
**Pull request by raducrisan1**

### CODE REVIEW

File: src/master/develop/deleting.md [hunk 1]
```diff
@@ -18,6 +18,31 @@ This feature is not yet supported or not documented.
...
+
+var client = ImmuClient.NewBuilder().WithServerUrl(immudbServerAddress).Build();
+await client.Open("immudb", "immudb", "defaultdb");
+
+string key = "hello";
+
+try
+{
+    await client.VerifiedSet(key, "immutable world!");
+    await client.Delete("key");
```
Comment (byo) [inline src/master/develop/deleting.md:33]:
```suggestion
    await client.Delete(key);
```

File: src/master/connecting/sdks.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ The most well-known and recommended immudb SDK is written in [Golang](https://go
 | `go`               | immudb team  | 1.3.2       |     [link](https://pkg.go.dev/github.com/codenotary/immudb/pkg/client)  |                                   |
 | `python`               | immudb team  | 1.3.2       |  [link](https://github.com/codenotary/immudb-py) |                                     |
 | `JAVA`               | immudb team  | 1.3.2       |   [link](https://github.com/codenotary/immudb4j)  |                                      |
+| `.NET`               | immudb team  | 1.3.2       |   [link](https://github.com/codenotary/immudb4net)  |                                      |
```
Comment (byo) [inline src/master/connecting/sdks.md:22]:
```suggestion
| `.NET`               | immudb team  | 1.4.0       |   [link](https://github.com/codenotary/immudb4net)  |                                      |
```

(and I should have updated the go version :facepalm:)

File: src/master/develop/queries-history.md [hunk 3]
```diff
@@ -650,9 +740,9 @@ const cl = new ImmudbClient({ host: IMMUDB_HOST, port: IMMUDB_PORT });
 ```
 :::
 
-::: tab .Net
+::: tab .NET
 This feature is not yet supported or not documented.
-Do you want to make a feature request or help out? Open an issue on [.Net sdk github project](https://github.com/codenotary/immudb4dotnet/issues/new)
```
Comment (byo) [inline src/master/develop/queries-history.md:655]: I think this is actually supported?

File: src/master/develop/reading.md [hunk 4]
```diff
@@ -351,6 +382,11 @@ This feature is not yet supported or not documented.
 Do you want to make a feature request or help out? Open an issue on [Java sdk github project](https://github.com/codenotary/immudb4j/issues/new)
 :::
 
+::: tab .NET
+This feature is not yet supported or not documented.
```
Comment (byo) [inline src/master/develop/reading.md:386]: I think I missed that in SDK - just asking for confirmation that we don't support GetAtRevision there.

File: src/master/develop/queries-history.md (hunk 3 above)
Comment (raducrisan1) [reply src/master/develop/queries-history.md:655]: VerifiableSetReference is not added. I was too much inspired by Java, I missed that.  I can add it soon; if you think it's an important one then I can start adding it.

File: src/master/develop/reading.md (hunk 4 above)
Comment (raducrisan1) [reply src/master/develop/reading.md:386]: It is there. Adding the sample now.

File: src/master/develop/queries-history.md (hunk 3 above)
Comment (raducrisan1) [reply src/master/develop/queries-history.md:655]: added

File: src/master/connecting/sdks.md (hunk 2 above)
Comment (raducrisan1) [reply src/master/connecting/sdks.md]: done

**Review by byo — APPROVED** [review]

**Pull request merged**